test/v2/services/AmendUkPropertyPeriodSummaryServiceSpec.scala

```diff
@@ -120,17 +120,17 @@ class AmendUkPropertyPeriodSummaryServiceSpec extends ServiceSpec {
     "unsuccessful" should {
       "map errors according to spec" when {
```

Comment:
As with the ConnectorSpec, could the test and requestBody etc be stuffed down to the end of the test class?